cw-orch-daemon/src/keys/private.rs

```diff
@@ -168,15 +167,14 @@ impl PrivateKey {
     // Generate private key from Phrase
     fn gen_private_key_phrase<C: secp256k1::Signing + secp256k1::Context>(
         secp: &Secp256k1<C>,
-        phrase: Phrase,
+        phrase: bip39::Mnemonic,
         account: u32,
         index: u32,
         coin_type: u32,
         seed_phrase: &str,
```

Comment:
We're calling this the seed_phrase but it looks like it's the password that can be used to generate the private key, allongside the seed_phrase.

Reply:
It's called a passphrase, so renaming the variable